Source/Scene/ImageryLayerCollection.js

```diff
@@ -325,6 +325,105 @@ ImageryLayerCollection.prototype.lowerToBottom = function (layer) {
 
 var applicableRectangleScratch = new Rectangle();
 
+/**
+ * Determines the imagery layers that are intersected by a pick ray. To compute a pick ray from a
+ * location on the screen, use {@link Camera.getPickRay}.
```

Comment:
I posted some caveats about the rectangle-based approach in https://github.com/CesiumGS/cesium/issues/8692. This function is still helpful in a lot of cases but the limitations should be documented.